website/rss.py

```diff
@@ -0,0 +1,23 @@
+import feedparser
+import eventlet
+
+
+with eventlet.Timeout(10):
+    url = 'http://feeds.feedburner.com/PythonInsider/.rss'
+
+
+def get_rss():
+
+    feed = feedparser.parse(url)
+    posts_to_show = []
+
+    for post in feed.entries[0:4]:
+        title = post.title
+        link = post.link
+        posts_to_show.append((title, link))
+    if not posts_to_show:
+        posts_to_show = [('Click here for latest news from python.org',
+                               'https://pythoninsider.blogspot.com/')]
+        return posts_to_show
+    else:
+        return posts_to_show
```

Comment:
```
if not posts:
   posts = [default values]
return posts
```
This exactly what I meant in the previous comment that "it will break"
if entries is empty then only posts will be []
But then you are doing list slicing on entries [0:4]
which will break in some cases!